src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -103,23 +103,22 @@ protected virtual JObject InvokeFunction(JArray _params)
             UInt160 script_hash = UInt160.Parse(_params[0].AsString());
             string operation = _params[1].AsString();
             ContractParameter[] args = _params.Count >= 3 ? ((JArray)_params[2]).Select(p => ContractParameter.FromJson(p)).ToArray() : new ContractParameter[0];
-            UInt160 sender = _params.Count >= 4 ? AddressToScriptHash(_params[3].AsString()) : null;
-            Signers signers = _params.Count >= 5 ? SignersFromJson((JArray)_params[4]) : null;
+            Signers signers = _params.Count >= 4 ? SignersFromJson((JArray)_params[3]) : null;
+
             byte[] script;
             using (ScriptBuilder sb = new ScriptBuilder())
             {
                 script = sb.EmitAppCall(script_hash, operation, args).ToArray();
             }
-            return GetInvokeResult(script, sender, signers);
+            return GetInvokeResult(script, signers);
         }
 
         [RpcMethod]
         protected virtual JObject InvokeScript(JArray _params)
         {
-            byte[] script = Convert.FromBase64String(_params[0].AsString());
-            UInt160 sender = _params.Count >= 2 ? AddressToScriptHash(_params[1].AsString()) : null;
-            Signers signers = _params.Count >= 3 ? SignersFromJson((JArray)_params[2]) : null;
-            return GetInvokeResult(script, sender, signers);
+            byte[] script = _params[0].AsString().HexToBytes();
```

Comment:
The modification of https://github.com/neo-project/neo-modules/pull/362 goes back here 🤔